src/View/Helper/HtmlHelper.php

```diff
@@ -344,7 +344,14 @@ public function link($title, $url = null, array $options = [])
             $title = h(urldecode($title));
             $escapeTitle = false;
         }
-
+        
+        /* A "simple phishing" vulnerability
+         * Consider adding a configuration option to disable?
+         */ https://dev.to/ben/the-targetblank-vulnerability-by-example
+        if (isset($options['target'])) {
+            $options['rel'] = 'noopener noreferrer';
+        }
+        
```

Comment:
This will override user's defined `rel` in certain cases (i.e. bookmark, author, etc.).